satellite/satellitedb/usagerollups.go

```diff
@@ -293,29 +298,24 @@ func (db *usagerollups) GetBucketTotals(ctx context.Context, projectID uuid.UUID
 
 		bucketUsage.Egress = memory.Size(totalEgress).GB()
 
-		bucketStorageTallies, err := storageQuery(ctx,
-			dbx.BucketStorageTally_ProjectId([]byte(projectID.String())),
-			dbx.BucketStorageTally_BucketName([]byte(bucket)),
-			dbx.BucketStorageTally_IntervalStart(since),
-			dbx.BucketStorageTally_IntervalStart(before))
-
+		storageRow := db.db.QueryRowContext(ctx, storageQuery, []byte(projectID.String()), []byte(bucket), since, before)
 		if err != nil {
 			return nil, err
 		}
 
-		// fill metadata, objects and stored data
-		// hours calculated from previous tallies,
-		// so we skip the most recent one
-		for i := len(bucketStorageTallies) - 1; i > 0; i-- {
-			current := bucketStorageTallies[i]
-
-			hours := bucketStorageTallies[i-1].IntervalStart.Sub(current.IntervalStart).Hours()
-
-			bucketUsage.Storage += memory.Size(current.Remote).GB() * hours
-			bucketUsage.Storage += memory.Size(current.Inline).GB() * hours
-			bucketUsage.ObjectCount += float64(current.ObjectCount) * hours
+		var inline, remote int64
+		var objectCount uint64
+		err = storageRow.Scan(&inline, &remote, &objectCount)
+		if err != nil {
+			if err != sql.ErrNoRows {
+				return nil, err
+			}
 		}
 
+		// fill storage and object count
+		bucketUsage.Storage = memory.Size(inline + remote).GB()
```

Comment:
should this still be done if `err == sql.ErrNoRows`?

Reply:
yeah, it will return zero